theming/_variables.scss

```diff
@@ -175,13 +175,13 @@ footer {
 }
 
 a {
-  color: rgba(0, 0, 0, 0.6) !important;
+  color: rgb(0, 68, 170) !important;
   background-color: transparent;
   text-decoration: none;
 }
```

Comment:
We can go ahead and just delete the anchor tag styling for now since they're blue by default (we make a non-global/non-tag level selector if we want certain links to have no text-decoration, etc.)